<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdateBalanceDialog</class>
 <widget class="QWidget" name="UpdateBalanceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UpdateBalanceDialog</string>
  </property>
  <widget class="QPushButton" name="btn_AddMoney">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Cancel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Отмена</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Balance">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>231</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Введите сумму пополнения баланса</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_AddMoney">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
